datatypes/src/raster/raster_properties.rs

```diff
@@ -34,9 +75,14 @@ impl RasterProperties {
     }
 
     pub fn string_property(&self, key: &RasterPropertiesKey) -> Result<String> {
+        if key.domain.is_none() && key.key == "band_name" {
```

Comment:
why is this the way it is, what does this mean? :D

Reply:
the key is a pair and there are fixed value fields in the properties. To be able to address them i added this. we could also remove the fixed keys